src/components/accordion/AccordionItem.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Add justification for disabling or remove

Reply:
Accessbility complaint by eslint. This is dawid code